games-arcade/rocksndiamonds/rocksndiamonds-3.3.1.2.recipe

```diff
@@ -56,6 +56,9 @@ BUILD()
 
 INSTALL()
 {
+	addResourcesToBinaries $portDir/additional-files/rocksndiamonds.rdef \
+		$binDir/rocksndiamonds
```

Comment:
I haven't tested it yet, but it looks like you need to move these two lines:
```
	addResourcesToBinaries $portDir/additional-files/rocksndiamonds.rdef \
		$binDir/rocksndiamonds
```
to the end of the INSTALL function (or at least after the call to cp) and replace, in the second line, the $binDir by $destdir:
```
	addResourcesToBinaries $portDir/additional-files/rocksndiamonds.rdef \
		$destdir/rocksndiamonds
```

BTW, you also need to add
```
ADDITIONAL_FILES="
	rocksndiamonds.iom
	rocksndiamonds.rdef
	"
```
after the PATCHES line.